package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/rpx-xui-common-lib",
-  "version": "1.7.49-update-from-r3.5-to-4-v3",
+  "version": "1.7.47-callback-copy-user-v1",
```

Comment:
Preferable to have same version as base branch version but different tag

Reply:
Updated.